### ISSUE TITLE
Uncaught TypeError: Expected an object with 'start' and 'end' properties.

### ISSUE THREAD

**Issue by shgysk8zer0** [mmk2410/atom-oss-license#3]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.2 x64
**Electron**: 1.6.9
**OS**: "Fedora release
**Thrown From**: [atom-oss-license](https://github.com/mmk2410/atom-oss-license) package 1.1.1

### Stack Trace

Uncaught TypeError: Expected an object with 'start' and 'end' properties.

```
At /home/shgysk8zer0/.atom/packages/atom-oss-license/lib/atom-oss-license.coffee:285

TypeError: Expected an object with 'start' and 'end' properties.
    at Object.hasCopyright (/packages/atom-oss-license/lib/atom-oss-license.coffee:285:29)
    at Object.hasCopyright (/packages/atom-oss-license/lib/atom-oss-license.coffee:283:13)
    at Object.hasCopyright (/packages/atom-oss-license/lib/atom-oss-license.coffee:283:13)
    at Object.addLicense (/packages/atom-oss-license/lib/atom-oss-license.coffee:319:13)
    at Object.lgpl_v3 (/packages/atom-oss-license/lib/atom-oss-license.coffee:237:6)
    at /packages/atom-oss-license/lib/atom-oss-license.coffee:69:31)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:265:35)
    at /usr/share/atom/resources/app/src/command-registry.js:3:65
    at Object.didConfirmSelection (/usr/share/atom/resources/app/node_modules/command-palette/lib/command-palette-view.js:106:35)
    at SelectListView.confirmSelection (/usr/share/atom/resources/app/node_modules/atom-select-list/src/select-list-view.js:313:26)
    at SelectListView.didClickItem (/usr/share/atom/resources/app/node_modules/atom-select-list/src/select-list-view.js:228:16)
    at ListItemView.onclick (/usr/share/atom/resources/app/node_modules/atom-select-list/src/select-list-view.js:168:37)
    at ListItemView.didClick (/usr/share/atom/resources/app/node_modules/atom-select-list/src/select-list-view.js:361:16)
```

### Commands

```
     -2:52.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -2:19.2.0 core:select-all (input.hidden-input)
  2x -2:14.4.0 core:backspace (input.hidden-input)
     -1:40.6.0 core:move-right (input.hidden-input)
     -1:14.9.0 core:paste (input.hidden-input)
     -1:10.8.0 core:backspace (input.hidden-input)
     -1:09.8.0 core:select-all (input.hidden-input)
     -1:09.5.0 core:backspace (input.hidden-input)
 13x -0:49.8.0 core:move-left (input.hidden-input)
     -0:48.5.0 core:move-right (input.hidden-input)
  6x -0:48.1.0 core:backspace (input.hidden-input)
     -0:31.2.0 intentions:highlight (input.hidden-input)
     -0:30.3.0 settings-view:open (input.hidden-input)
     -0:15.7.0 intentions:highlight (input.hidden-input)
     -0:15.4.0 command-palette:toggle (input.hidden-input)
     -0:06.4.0 License: LGPL v3 (input.hidden-input)
```

### Non-Core Packages

```
atom-autocomplete-php 0.25.6 
atom-live-server 2.1.2 
atom-oss-license 1.1.1 
busy-signal 1.4.3 
color-picker 2.2.5 
docblockr 0.11.0 
file-icons 2.1.11 
gist-it 0.9.2 
gtk-dark-theme 0.1.4 
intentions 1.1.5 
language-apache 1.7.0 
language-ini 1.19.0 
linter 2.2.0 
linter-csslint 1.3.4 
linter-eslint 8.2.1 
linter-php 1.3.2 
linter-ui-default 1.6.7 
merge-conflicts 1.4.5 
minimap 4.29.6 
minimap-autohide 0.10.1 
minimap-git-diff 4.3.1 
minimap-linter 2.0.2 
pandoc-convert 1.1.1 
platformio-ide-terminal 2.5.5 
```

**Comment by eggbit** [comment]
Any update to this? Getting the same error with 1.21.0-beta0.

**Comment by mmk2410** [comment]
Thanks for reporting and sorry that it took so long. The bug should be fixed in [v1.1.2](https://github.com/mmk2410/atom-oss-license/releases/tag/v1.1.2)
